boxoffice/forms/order.py

```diff
@@ -3,7 +3,7 @@
 from baseframe import __
 import baseframe.forms as forms
 
-__all__ = ['LineItemForm', 'BuyerForm', 'OrderSessionForm']
+__all__ = ['LineItemForm', 'BuyerForm', 'OrderSessionForm', 'OrderRefundForm']
 
 
 def trim(length):
```

Comment:
Unrelated, but can we move `trim` into `baseframe.forms.filters` now that that module exists?